go/vt/tabletserver/config.go

```diff
@@ -20,6 +21,9 @@ import (
 	topodatapb "github.com/youtube/vitess/go/vt/proto/topodata"
 )
 
+// longTime is for setting long timeouts, practically forever.
+const longTime = 24 * 365 * 10 * time.Hour
```

Comment:
I think I'd rather use 0 in this case, and in execRequest have:
if timeout != 0 {
  var cancel context.CancelFunc
  ctx, cancel = context.WithTimeout(timeout)
  defer cancel()
}

What do you think?